src/goto-symex/goto_symex_state.h

```diff
@@ -138,10 +139,17 @@ class goto_symex_statet final : public goto_statet
     return threads[source.thread_nr].call_stack;
   }
 
-  /// Create a new instance of the L0-renamed object \p ssa. As part of this,
-  /// turn \p ssa into an L1-renamed SSA expression. When doing so, use
-  /// \p l1_index as the L1 index, which must not previously have been used.
-  void add_object(ssa_exprt &ssa, std::size_t l1_index, const namespacet &ns);
+  /// Create a new instance of the object \p expr. Such an instance is an
+  /// L1-renamed SSA expression such the L1-index of which has not previously
+  /// been used.
```

Comment:
This sentence is not clear, should it be "such that its L1-index has note previously been used"